Makefile

```diff
@@ -1,21 +1,35 @@
-.PHONY: deps clean rocket test
-
-all: rocket
+all: deps
 
 rocket:
 	go install
 
+.PHONY: deps
 deps:
-	glide install
+	dep ensure
 
+.PHONY: clean
 clean:
-	rm rocket
+	rm -f rocket
+	pg_ctl -D /usr/local/var/postgres stop -s -m fast
```

Comment:
I don't think it's safe for a Make target to me shutting down Postgres on people's machines. I think that if you're going to shut down someone's Postgres you should at least make it obvious that you're doing so by making it a separate target like `shutdown-db`. With this setup your Postgres server gets shut down whenever your run tests.